Formula/orgalorg.rb

```diff
@@ -0,0 +1,26 @@
+class Orgalorg < Formula
+  desc "Parallel SSH commands executioner and file synchronization tool"
+  homepage "https://github.com/reconquest/orgalorg"
+  url "https://github.com/reconquest/orgalorg.git",
+      tag:      "1.0.1",
+      revision: "7ccd02c521f79e04720aa929e21715d46668162f"
+  license "MIT"
+  head "https://github.com/reconquest/orgalorg.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-mod=mod", "-ldflags", "-s -w -X main.version=#{version}", *std_go_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/orgalorg --version")
+    assert_match "orgalorg - files synchronization on many hosts.", shell_output("#{bin}/orgalorg --help")
+
+    port = free_port
+    output = shell_output("#{bin}/orgalorg -u tester -o 127.0.0.1:#{port} -C uptime 2>&1", 1)
+    assert_match(/connecting to cluster failed/, output)
+    assert_match(/dial tcp 127.0.0.1:#{port}: connect: connection refused/, output)
+    assert_match(/can't connect to address: \[tester@127.0.0.1:#{port}\]/, output)
```

Comment:
Since this is not using any regex features it can just compare a string.
```suggestion
    assert_match("connecting to cluster failed", output)
    assert_match("dial tcp 127.0.0.1:#{port}: connect: connection refused", output)
    assert_match("can't connect to address: [tester@127.0.0.1:#{port}]", output)
```